conf.d/sdk.fish

```diff
@@ -44,7 +47,7 @@ function __fish_sdkman_run_in_bash
              env | grep -i -E \"^(`echo \${SDKMAN_CANDIDATES_CSV} | sed 's/,/|/g'`)_HOME\" >> $pipe;
              echo \"SDKMAN_OFFLINE_MODE=\${SDKMAN_OFFLINE_MODE}\" >> $pipe;
              echo \"SDKMAN_ENV=\${SDKMAN_ENV}\" >> $pipe" # it's not an environment variable!
-    set bashDump (cat $pipe; rm $pipe)
+    set bashDump (cat $pipe; command rm $pipe)
```

Comment:
Can you please explain which problem this change solves? :thinking:

Reply:
Sorry I'm late with this! I have a separate command in my config that overrides `rm` (calls trash instead), which breaks this call. If I use the `command` to call `rm`, it seems to get the same original effect.

Reply:
@reitzig Any thoughts on this? I don't know of a more fish-y way to handle this particular command override 🤷

Reply:
I don't think it's feasible to look out for overrides of "basic" commands. :/ See also #23.